first-read-last-write-cache/src/cache.rs

```diff
@@ -427,4 +393,88 @@ mod tests {
             }
         }
     }
+
+    proptest! {
+        #[test]
+        fn test_merge_fuzz(s: u8) {
+            let mut testvec = vec![0; 15];
+            for i in 0..15 {
+                testvec[i] = ((s as u16 + i as u16) % 255) as u8;
```

Comment:
let's replace it with `testvec[i] = s.wrapping_add(i as u8)`

Reply:
Ohh, I just noticed another thing. Let's extract 15 to a variable in this function.